packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -75,6 +90,36 @@ const DEFAULT_CONTROLS = {
 	duotone: true,
 };
 
+const popoverProps = {
+	placement: 'left-start',
+	offset: 36,
+	shift: true,
+	className: 'block-editor-duotone-control__popover',
```

Comment:
It's not great when we use "classnames" as reusable components. In the ideal scenario, the styles should be defined in the same place where the classname is used.

Reply:
yes! this I think can be refactored when we work on using the filter panel for the toolbar controls, I will make a note to do just that